### PR TITLE
Various minor fixes

### LINKED ISSUE #192: Crash when opening the lexica botania (1.9)
I was able to do the tutorial part, but after that it crashed and I cannot open it now.

http://pastebin.com/LTFZBHw2

### PR DESCRIPTION
**Pull request by yrsegal**
separated from the burst fixer since this doesn't change functionality, just fix bugs.

Closes #192, #194, #196

### CODE REVIEW

File: src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java [hunk 1]
```diff
@@ -53,9 +53,10 @@ public ItemManasteelAxe() {
 
 	public ItemManasteelAxe(ToolMaterial mat, String name) {
 		super(mat, ImmutableSet.of());
-		// todo attack speed and set
 		setCreativeTab(BotaniaCreativeTab.INSTANCE);
 		setUnlocalizedName(name);
+		this.damageVsEntity = 9f;
+		this.attackSpeed = 0.9f;
```
Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java]: For those wondering, these are the vanilla values for an Iron axe.

File: src/main/java/vazkii/botania/common/item/equipment/tool/terrasteel/ItemTerraAxe.java [hunk 2]
```diff
@@ -76,6 +76,7 @@
 	public ItemTerraAxe() {
 		super(BotaniaAPI.terrasteelToolMaterial, LibItemNames.TERRA_AXE);
 		MinecraftForge.EVENT_BUS.register(this);
+		this.attackSpeed = 1f;
```
Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/equipment/tool/terrasteel/ItemTerraAxe.java]: This is the attack speed value for a Diamond Axe.

File: src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java [hunk 3]
```diff
@@ -53,9 +53,10 @@ public ItemManasteelAxe() {
 
 	public ItemManasteelAxe(ToolMaterial mat, String name) {
 		super(mat, ImmutableSet.of());
-		// todo attack speed and set
 		setCreativeTab(BotaniaCreativeTab.INSTANCE);
 		setUnlocalizedName(name);
+		this.damageVsEntity = 8f;
+		this.attackSpeed = -3.1f;
```
Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java]: These are the stats of an IronAxe.

File: src/main/java/vazkii/botania/common/item/equipment/tool/terrasteel/ItemTerraAxe.java [hunk 4]
```diff
@@ -76,6 +76,7 @@
 	public ItemTerraAxe() {
 		super(BotaniaAPI.terrasteelToolMaterial, LibItemNames.TERRA_AXE);
 		MinecraftForge.EVENT_BUS.register(this);
+		this.attackSpeed = -3f;
```
Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/equipment/tool/terrasteel/ItemTerraAxe.java]: This is the stat for a Diamond Axe.

File: src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java (hunk 3 above)
Comment (williewillus) [inline src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java:59]: supposed to be negative?

Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java:59]: yeah, the base is 4, so I have to offset it.

Comment (yrsegal) [inline src/main/java/vazkii/botania/common/item/equipment/tool/manasteel/ItemManasteelAxe.java:59]: Before it was speed 4, damage 4 (which allows spamclicking)

**Comment by yrsegal** [comment]
you'll apparently have to close the issues manually, since MC19 isn't your master branch

**Pull request merged**